packages/xchain-mayachain-amm/__mocks__/mayanode-api/mayanode-api.ts

```
@@ -35,5 +35,17 @@ export default {
       const resp = await importjson(`./responses/latestBlock.json`)
       return [200, resp]
     })
+    // Gas rates mock for ETH
+    mock.onGet(/api\.etherscan\.io.*gastracker/).reply(() => {
+      return [200, {
```

ESLint

[error] Replace `200,` with `⏎········200,⏎·······`

(prettier/prettier)